test/e2e/stock-time-series/stock-time-series.test.ts

```diff
@@ -155,3 +155,21 @@ describe('StockTimeSeries', () => {
     });
   });
 });
+
+describe('#quote', () => {
```

Comment:
please put this test inside the `describe('StockTimeSeries', () => { ` block, right after the `describe('#monthlyAdjusted', () => {` block, so the `quote `test can have access to the `api` object